Product/SoapUI_Test/FileUtils/src/test/java/nhinc/FileUtilsTest.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (c) 2009-2017, United States Government, as represented by the Secretary of Health and Human Services.
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *     * Redistributions of source code must retain the above
+ *       copyright notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the documentation
+ *       and/or other materials provided with the distribution.
+ *     * Neither the name of the United States Government nor the
+ *       names of its contributors may be used to endorse or promote products
+ *       derived from this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE UNITED STATES GOVERNMENT BE LIABLE FOR ANY
+ * DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package nhinc;
+
+import static nhinc.FileUtils.getCheckpoint;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+import java.io.File;
+import java.io.IOException;
+import java.text.MessageFormat;
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.mockito.internal.verification.AtLeast;
+
+/**
+ * @author Tran Tang
+ *
+ */
+public class FileUtilsTest {
+    private final Logger log = mock(Logger.class);
+    private static final String RESOURCE_FOLDER = "src/test/resources/FileUtils";
+    private static final String BACKUP_FOLDER = "src/test/resources/FileUtils/backupHere";
+    private static final String EXCHANGE_FILE = "exchangeInfo.xml";
+    private static final String EXCHANGE_FILE_ORG = "exchangeInfoOrg.xml";
+    private static final String TEST_FILE = "testFile.properties";
+    private static final String TEST_FILE_ORG = "testFileOrg.properties";
+
+    @BeforeClass
```

Comment:
Remove them if we don't use or have any additional logic

Reply:
currently i don't wanted the testFileOrg.properties to be modified; this get flagged as a modified files when we re-compiled the projects; so i copied the testFileOrg.properties to testFile.properties modified the testFile.properties and delete the testFile.properties